tests/spec/core/linter-rules/check-internal-slots-spec.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+describe("Core Linter Rule - 'check-internal-slots'", () => {
+  const ruleName = "check-internal-slots";
+  const config = {
+    lint: { [ruleName]: true },
+  };
+  let rule;
+  beforeAll(async () => {
+    rule = await new Promise(resolve => {
+      require([`core/linter-rules/${ruleName}`], ({ rule }) => resolve(rule));
+    });
+  });
+  const doc = document.implementation.createHTMLDocument("test doc");
+  beforeEach(() => {
+    // Make sure every unordered test get an empty document
+    // See: https://github.com/w3c/respec/pull/1495
+    while (doc.body.firstChild) {
+      doc.body.removeChild(doc.body.firstChild);
+    }
+  });
+
+  it("only returns an error when there is no '.' between var and an internal slot", async () => {
+    doc.body.innerHTML = `
+    <var>bar</var><a>[[foo]]</a>
```

Comment:
Small nit: could you kindly please indent these by two spaces? (similar to the ones below)